app/lib/api/file_system/file_system_remote.dart

```diff
@@ -144,9 +144,29 @@ mixin RemoteSystem {
 
   Future<void> clearCachedContent() async {
     var cacheDir = await getRemoteCacheDirectory();
+    print('Cache directory path: $cacheDir');
     var directory = Directory(cacheDir);
-    if (await directory.exists()) {
-      await directory.delete(recursive: true);
+    bool exists = await directory.exists();
```

Comment:
```suggestion
    final exists = await directory.exists();
```
like said in the pdf import pr, a final is better since it indicates that this value won't be changed